tools/dissector/sample_dissector.h

```diff
@@ -51,7 +51,9 @@ namespace OpenDDS
       {
       }
 
-      const char* what() const throw (){
+      ~Sample_Dissector_Error() {}
+
+      const char* what() const throw() override {
```

Comment:
`override` is from C++11.  If you can get away with leaving it off, that's probably the way to go.  If not we'll need an #ifdef to conditionally add `override`.

Reply:
I thought one of the builds complained about it, but I looked through them yesterday and I couldn't find it, they only complained about the destructor. I think I mixed them up and tried to fixed something that didn't need fixing.